### PR TITLE
Update breeze.d.ts to Breeze 1.5.2

### PR DESCRIPTION
**Pull request by steveschmitt**
Added missing methods and parameter options

### CODE REVIEW

**Comment by vvakame** [comment]
@basarat could you review this PR?

File: breeze/breeze.d.ts [hunk 1]
```diff
@@ -1,13 +1,15 @@
 // Type definitions for Breeze 1.4
 // Project: http://www.breezejs.com/
-// Definitions by: Boris Yankov <https://github.com/borisyankov/>, IdeaBlade <https://github.com/IdeaBlade/Breeze/>
```
Comment (basarat) [inline breeze/breeze.d.ts]: Why? Its good to know who to contact in case of difficulties

File: breeze/breeze.d.ts [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+// Definitions by: Boris Yankov <https://github.com/borisyankov/>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
 // Updated Jan 14 2011 - Jay Traband ( www.ideablade.com).
-// Updated March 27 2013 - John Lantz (www.ideablade.com).
```
Comment (basarat) [inline breeze/breeze.d.ts]: Curious? why removed?

**Comment by basarat** [comment]
Other than the minor commented name changes, the rest is pretty solid. Glad to see it become independent of Q.

File: breeze/breeze.d.ts (hunk 1 above)
Comment (steveschmitt) [inline breeze/breeze.d.ts]: Maybe I shouldn't have removed that.  The contacts for individual changes are in the changelog below, though.

File: breeze/breeze.d.ts (hunk 2 above)
Comment (steveschmitt) [inline breeze/breeze.d.ts]: John Lantz is no longer at ideablade, so I didn't want anyone to come looking for him.

File: breeze/breeze.d.ts (hunk 1 above)
Comment (basarat) [inline breeze/breeze.d.ts]: Cool. I'll add that after merge. Thanks!

**Pull request merged**